### PR TITLE
feat: delete and unify cosmos send form modal

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Second and last PR of the Cosmos send form unifying, entirely removing the older components and the branching.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

EVM, UTXO and Cosmos SDK sends could break, test accordingly - tested Cosmos sends still work with memo

## Testing

- EVM sends still work
- UTXO sends still work
- Cosmos SDK sends still work
  - Memos work - should be displayed as an input on asset selection / value input and as a row at confirm step 
- Max sends still work for all chains
- Fees are still correct for all chains

### Engineering

☝🏽 
- Removal looks sane and no file has been forgotten
- Cosmos-specific constants in the common send component don't look out of place and don't need a better home

### Operations

☝🏽

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2939** <a href="https://app.graphite.dev/github/pr/shapeshift/web/2939" target="_blank">Graphite</a> 
    * **PR #2986** <a href="https://app.graphite.dev/github/pr/shapeshift/web/2986" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/2986?utm_source=stack-comment).

File: src/components/Modals/Send/views/Details.tsx [hunk 1]
```diff
@@ -188,12 +214,67 @@ export const Details = () => {
...
+                  type='text'
+                  variant='filled'
+                  placeholder={translate('modals.send.sendForm.optionalAssetMemo', {
+                    assetSymbol: asset.symbol,
+                  })}
+                />
+              )}
+            />
+          </FormControl>
+        )}
```
Comment (gomesalexandre) [inline src/components/Modals/Send/views/Details.tsx:266]: Taken from the deleted cosmos `<Details />`

File: src/components/Modals/Send/views/Details.tsx [hunk 2]
```diff
@@ -69,6 +85,16 @@ export const Details = () => {
...
 
+  const showMemoField = useMemo(
+    () => Boolean(asset && fromAssetId(asset.assetId).chainNamespace === CHAIN_NAMESPACE.CosmosSdk),
+    [asset],
+  )
+  const remainingMemoChars = useMemo(
+    () => bnOrZero(MAX_COSMOS_SDK_MEMO_LENGTH - Number(memo?.length)),
+    [memo],
+  )
+  const memoFieldError = remainingMemoChars.lt(0) && 'Characters Limit Exceeded'
```
Comment (0xApotheosis) [inline src/components/Modals/Send/views/Details.tsx:96]:
```suggestion
  const memoFieldError = remainingMemoChars.lt(0) && 'Character Limit Exceeded'
```

**Review by 0xApotheosis — APPROVED** [review]
LGTM!

- Tested that UTXO, EVM and CosmosSDK chains still send as expected.
- Memo worked when sending Cosmos.
- Cosmos SDK fee estimate reflected the actual fee.

File: src/components/Modals/Send/views/Details.tsx (hunk 2 above)
Comment (0xApotheosis) [reply src/components/Modals/Send/views/Details.tsx:96]: Non-blocking, resolving.

**Comment by 0xApotheosis** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/2986?utm_source=stack-comment).


Job Reference: 5RTfzUPQ8wDzu6QclVcQ

**Comment by 0xApotheosis** [comment]
Cypress is flaking at the moment, so I'll push this one through.

**Comment by tshifty** [comment]
ETH leaves some dust behind from a MAX send but besides that everything worked perfectly 👍

**Pull request merged**